shellclear/src/config.rs

```diff
@@ -28,6 +28,17 @@ impl Default for Config {
     }
 }
 
+impl From<Option<&str>> for Config {
+    fn from(config_dir_path_option: Option<&str>) -> Self {
```

Comment:
Naming: you can just use `config_dir` as the parameter name

Reply:
@jondot thanks for your review.
I updated the variable name and fixed the linting issue by running `cargo fmt` as you suggested.
I wonder if I can auto-format rust code in `vscode` (I installed `rust-analyzer` plugin).

Reply:
Yes, add the following

Format all languages on save:
```json
  "editor.formatOnSave": true,
```

For rust, use the `rust-analyzer` formatter:

```json
 "[rust]": {
    "editor.defaultFormatter": "rust-lang.rust-analyzer"
  },
```

Reply:
I also briefly updated the README file.

Reply:
Thanks, rust code is now auto-formatted on `vscode` :laughing: